amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -540,6 +540,66 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/file-plans/{filePlanId}/roles':
+    get:
+      tags:
+        - file-plans
+      summary: Get roles in a file plan
+      description: |
+        Gets a list of roles for the specified file plan **filePlanId**.
+        
+        You can use the **include** parameter to return additional information.
+        
+        The **where** parameter can also be used to filter by **personId**, **systemRoles** and **capabilityName**.
+
+        You can use the **where** parameter to 
+        * filter roles by user:
+        `where=(personId='admin')`
+        
+        * not include system roles in the results:
+        `where=(systemRoles=false)`
+        
+        * filter roles by specified capabilities:
+        `where=(capabilityName in ('AddToHold', 'ViewRecords'))`
+        
+        This may be combined with all filter, as shown below:
+        
+        `where=(systemRoles=false and personId='johndoe' and capabilityName in ('AddToHold', 'ViewRecords'))`
```

Comment:
@damianujma I have one question - "ViewRecords" capability is regarding to "Records Management" library right?

<img width="1517" alt="image" src="https://github.com/user-attachments/assets/5ecaae9c-8c5b-4ea7-b4f6-189dfc433a53" />

So that capability checks if you can see or not that library? (It's basically fix for blocker of https://hyland.atlassian.net/browse/ACS-9377)

Reply:
Yes, that capability checks if you can see that library.